libs/sdk-ui-dashboard/src/presentation/topBar/shareIndicators/shareStatus/ShareStatusIndicator.tsx

```diff
@@ -16,8 +16,13 @@ export const ShareStatusIndicator = (props: IShareStatusProps): JSX.Element => {
     } else {
         icon = "gd-icon-invisible";
         text = "header.shareStatus.private.text";
-        tooltip = "header.shareStatus.private.tooltip";
+        if (!props.isUnderStrictControl) {
```

Comment:
If you would switch if/else branches, you don't need to use unnecessary negation here.